### ISSUE TITLE
Cannot toggle windows on Mac using "non-standard" keyboard layouts (eg British - PC; French)

### ISSUE THREAD

**Issue by alecgibson** [electron/electron#17287]
* **Electron Version:**
  *  4.0.7
* **Operating System:**
  *  macOS 10.13.6
* **Last Known Working Electron version:**
  *  Can potentially be inferred from https://github.com/Microsoft/vscode/issues/68217

### Expected Behavior

Normally on MacOS, you can toggle between two windows of the same app with ⌘ + ` (Cmd + backtick). I have verified that this works in other apps installed on my machine (eg Chrome).

I expect to be able to do the same with Electron windows.

### Actual Behavior

Using Cmd + Backtick toggles windows using the "British" keyboard layout, but _not_ the "British - PC" layout.
### To Reproduce

1. Use macOS
2. Clone this repo: https://github.com/alecgibson/electron-quick-start

```sh
git clone https://github.com/alecgibson/electron-quick-start
cd electron-quick-start
npm install
npm start
```

3. Enable the "British" keyboard layout
4. Start the app 
5. Use Cmd + ` to toggle between the two spawned windows
6. Switch keyboard layout to "British - PC"
7. Try to use Cmd + ` again, but note that the windows no longer toggle (NB - the backtick key will have moved; verify you've got the right key by trying to type it somewhere)

For completeness, here is the simple code in the repo:

```javascript
const { app, BrowserWindow } = require('electron')

const windows = new Set();

function createWindow(url) {
  const window = new BrowserWindow({ width: 800, height: 600 });
  window.loadURL(url);
  windows.add(window);
  window.on('closed', () => windows.delete(window));
}

app.on('ready', () => {
  createWindow('https://electronjs.org');
  createWindow('https://github.com/electron/electron-quick-start');
});
```

### Additional Information

This was first raised as an issue on VS Code, but it looks like it's an Electron bug: https://github.com/Microsoft/vscode/issues/68217

**Comment by bengourley** [comment]
The last working version appears to be `2.0.18`. Every version of Electron that I tried after that (starting with `v3.0.0`) cmd+backtick does not work.

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by alecgibson** [comment]
I've just tested this in v8.0.1, and it's still broken. Is this on anyone's radar? It's a really annoying bug.

**Comment by erralb** [comment]
Hi, I have the same problem on a MacBook pro 16 with Catalina and a French keyboard (using VSCode)

**Comment by titouanmathis** [comment]
Hi, 

Same bug for me on a Macbook Pro 13" 2017 with macOS Catalina and a French keyboard, using VSCode 1.45.1 and Electron 7.2.4.

**Comment by cogat** [comment]
I am here hoping for a fix that helps with window switching on the Trello app. I use Dvorak keyboard layout, but switching between Dvorak and US layout doesn't seem to make a difference (backtick is in the same place on both). When I run @alecgibson's example I can't immediately switch windows with CMD+`, but after waiting about 10 seconds I suddenly can. I'm running MBP 2019 with Catalina.

**Comment by clavin** [comment]
It seems I fixed this issue in #27701. It took me a moment to find \` on this keyboard layout but I ran the example in this thread, confirmed it's still broken, then ran the example on a build with the fix from #27701 in it and Cmd + \` in the "British - PC" keyboard layout worked as well as the "French" keyboard layout.

**Comment by MarshallOfSound** [comment]
Fixed in #27701 🥳
